packages/app/src/app/components/WorkspaceSelect/WorkspaceSelect.tsx

```diff
@@ -110,7 +110,11 @@ export const WorkspaceSelect: React.FC<WorkspaceSelectProps> = React.memo(
                   }
                 >
                   <TeamAvatar
-                    avatar={team.avatarUrl}
+                    avatar={
+                      team.id === state.personalWorkspaceId && user
```

Comment:
In my last PR I didn't realise I needed to change the workspace select dropdown as well (it was still showing the blank letter avatar)

Reply:
Ooh yes, good catch!